desc/objectives/objective_funs.py

```diff
@@ -1072,24 +1090,37 @@ def jvp_unscaled(self, v, x, constants=None):
         """
         return self._jvp(v, x, constants, "compute_unscaled")
 
-    def print_value(self, *args, **kwargs):
+    def print_value(self, val, val0=None, **kwargs):
```

Comment:
I'd prefer something like `args, args0` instead of `val`, as `val` makes it sound like you are printing that value, when really these are the arguments to the objective compute function